county_health_preprocessing_xl.ipynb

```diff
@@ -0,0 +1,4188 @@
+       "    <tr>\n",
+       "      <th>15</th>\n",
+       "      <td>AL</td>\n",
+       "      <td>Autauga County</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>2004-2006</td>\n",
+       "      <td>Violent crime rate</td>\n",
+       "      <td>43.0</td>\n",
+       "      <td>289.5827344</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>16</th>\n",
+       "      <td>AL</td>\n",
+       "      <td>Autauga County</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>2005-2007</td>\n",
+       "      <td>Violent crime rate</td>\n",
+       "      <td>43.0</td>\n",
+       "      <td>255.9350455</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>17</th>\n",
+       "      <td>AL</td>\n",
+       "      <td>Autauga County</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>2006-2008</td>\n",
+       "      <td>Violent crime rate</td>\n",
+       "      <td>43.0</td>\n",
+       "      <td>255.7833615</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>18</th>\n",
+       "      <td>AL</td>\n",
+       "      <td>Autauga County</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>2007-2009</td>\n",
+       "      <td>Violent crime rate</td>\n",
+       "      <td>43.0</td>\n",
+       "      <td>290.5093729</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   state          county  state_code  county_code  year_span  \\\n",
+       "14    AL  Autauga County         1.0          1.0  2003-2005   \n",
+       "15    AL  Autauga County         1.0          1.0  2004-2006   \n",
+       "16    AL  Autauga County         1.0          1.0  2005-2007   \n",
+       "17    AL  Autauga County         1.0          1.0  2006-2008   \n",
+       "18    AL  Autauga County         1.0          1.0  2007-2009   \n",
+       "\n",
+       "          measure_name  measure_id    raw_value  \n",
+       "14  Violent crime rate        43.0  303.6262884  \n",
+       "15  Violent crime rate        43.0  289.5827344  \n",
+       "16  Violent crime rate        43.0  255.9350455  \n",
+       "17  Violent crime rate        43.0  255.7833615  \n",
+       "18  Violent crime rate        43.0  290.5093729  "
+      ]
+     },
+     "execution_count": 101,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "xldf = df_final.copy()\n",
+    "xldf.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 102,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(278285, 8)"
+      ]
+     },
+     "execution_count": 102,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "xldf = xldf[xldf.raw_value>0]\n",
```

Comment:
Curious why you're removing raw_value>0 here? Did you confirm 0 is not meaningful for the measures that have it?